<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushSettingsWidget</class>
 <widget class="QWidget" name="BrushSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrushSettingsWidget</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="radiusSpinBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="radiusSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="outerRadiusLabel">
     <property name="text">
      <string>Outer Radius</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="outerRadiusSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="outerRadiusSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="innerRadiusLabel">
     <property name="text">
      <string>Inner Radius</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="innerRadiusSpinBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QSlider" name="innerRadiusSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="rampPowerLabel">
     <property name="text">
      <string>Ramp Power</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="rampPowerSpinBox"/>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="rampPowerSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
